lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+require 'yaml'
+
+class Menu
+  def initialize(filename = 'menu.yaml')
+    @filename = filename
+  end
+
+  def display_menu
+    yaml_load
+  end
+
+  def has_dish?(dish)
+    display_menu.has_key? dish
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)